src/main/resources/templates/index.html

```diff
@@ -11,29 +11,29 @@
             width: 100%;
             padding: 16px 32px;
             text-alight: center;
-            font-size: 24px;
+            font-size: 25px;
         }
     </style>
 </head>
 <body>
-    <div class="container-fluid">
+    <section class="container-fluid">
         <div class="row">
-            <div class="col-md-12">
+            <div class="col-md-20">
                 <h1 class="text-center">
                     TimeKeeper Login Page
                 </h1>
                 <form role="form">
                     <div class="form-group">
-                        <label for="usernameInput">
+                        <label for="UsernameInput">
                             Username
                         </label>
-                        <input type="text" class="form-control" id="usernameInput" />
+                        <input type="text" class="form-control" id="UsernameInput" required />
                     </div>
                     <div class="form-group">
                         <label for="passwordInput">
                             Password
                         </label>
-                        <input type="password" class="form-control" id="passwordInput" />
+                        <input type="password" class="form-control" id="passwordInput" required="reuqired"/>
```

Comment:
I don't particularly agree with the naming change for the label. Plus, some of the changes to the input tags are either incorrect or incomplete.